otx/algorithms/anomaly/tasks/inference.py

```diff
@@ -70,11 +70,12 @@
 class InferenceTask(IInferenceTask, IEvaluationTask, IExportTask, IUnload):
     """Base Anomaly Task."""
 
-    def __init__(self, task_environment: TaskEnvironment) -> None:
+    def __init__(self, task_environment: TaskEnvironment, output_path: Optional[str] = None) -> None:
```

Comment:
Doesn't it matter if this task is done in common with other tasks? (nncf, etc.)
It seems strange that the init parameters are different for each task.

Reply:
In the perspective of change to make some error, it's ok. because it's an optional argument. I think there won't any error.
In the perspective of design, I think that it's rather natural to have different arguments to different task.
For example, user may need to save some output in inference task or training task but doesn't need to save some output from training in nncf.
maybe I miss some point of view. If you have, then please let me know.

Reply:
As far as I know, pot or nncf also creates a log folder and new model weights that there is an output.
Also, doesn't the addition of this parameter mean to replace the existing `/tmp/`?

Reply:
I see. I'll also add new argument to other tasks not updated yet.